### ISSUE TITLE
Can't use ssml -- Error: string required by Text constructor

### ISSUE THREAD

**Issue by LukeJanissen** [dialogflow/dialogflow-fulfillment-nodejs#159]
Whenever Im using SSML output in my Dialogflow output (V2),
I'm receive the following error: 
`Error: string required by Text constructor.  at new Text (/text-response.js:48:13)`

**Comment by sarahdwyer** [comment]
@LukeJanissen Can you provide your code and more information?

**Comment by LukeJanissen** [comment]
@sarahdwyer Of course! Whenever I'm using SSML code in the audio output section of the Google Assistant response in Dialogflow, my webhook returns the error I mentioned above. This happens at the moment I'm calling `new WebhookClient({request: request, response: response})' (so, whenever I'm using the Dialogflow SDK). I already stripped my entire code, so the only thing it would do is send a exact copy of the request back. This works as long as I'm not calling `new WebhookClient`, but I will need the SDK to add certain functionality to my webhook.

**Comment by sarahdwyer** [comment]
@LukeJanissen Do you have text-to-speech functionality and beta features enabled from settings?

**Comment by matthewayne** [comment]
Please provide the code throwing the error

**Comment by ivanchuk** [comment]
I have the same issue, the error happends without my code, just when `new WebhookClient({request: request, response: response})` is invoked (first line of webhook fulfillment. What can I do?

**Comment by dhAlcojor** [comment]
I'm having the same problem. According to Firebase logs the problem is nowhere in my code
![image](https://user-images.githubusercontent.com/567687/57287719-6a3a2680-70b8-11e9-83e4-e86cc81ae4fb.png)

**Comment by ivanchuk** [comment]
Solved! The problem start in Dialogflow. Go to your intent and remove Google Assistant responses you've configured. I don't know why but the Google Assistant payloads and v2 agent are incompatibles right now.

If you remove it, payload will fix the issue and Webhook constructor works fine!
